boto/iam/connection.py

```diff
@@ -1722,7 +1722,7 @@ def get_credential_report(self):
         params = {}
         return self.get_response('GetCredentialReport', params)
 
-    def create_virtual_mfa_device(self, path, device_name):
+    def create_virtual_mfa_device(self, device_name, path='/'):
```

Comment:
What's the strong reason for switching these? Not saying we shouldn't but I guess Boto has it implemented this way by default so it seems like an odd change. Of course we are moving away from Boto to Boto3 perhaps so its not a huge deal, but normally, I wouldn't agree with this.

Reply:
Ok. The caller(iam.py) has 'path' variable default as '/' and calls this API with that. So this change was not necessary.
Back to the original one. https://github.com/cloudian/boto/commit/6d7e94522d14d710fddd350d226f257ca98913c3